### ISSUE TITLE
Command line argument --gruntfile

### ISSUE THREAD

**Issue by jarrodpayne** [sindresorhus/grunt-concurrent#17]
**tl;dr** - This plugin doesn't respect `--gruntfile` grunt-cli argument (flag).

I am using this task as a part of Yeoman. My requirements are to use another Gruntfile.js, I'm calling `altGruntfile.js` here. 

This command `grunt build --gruntfile altGruntfile.js` fails when I remove the standard _Gruntfile.js_ when it gets to: `Running "concurrent:dist" (concurrent) task`

```
Running "concurrent:dist" (concurrent) task
----------------------------------------------------------------
    Warning: A valid Gruntfile could not be found. Please see the getting started guide for
    more information on how to configure grunt: http://gruntjs.com/getting-started
    Fatal error: Unable to find Gruntfile. Use --force to continue.

        Aborted due to warnings.
```

When the standard _Gruntfile.js_ remains, the `concurrent:dist` seems to use the settings it rather than using the settings from _altGruntfile.js_ even when running the command: 

`grunt build --gruntfile altGruntfile.js`

---

BTW, when I remove `concurrent:dist` from `build` task, and replace with tasks that should be run concurrently, it works as expected.
